### PR TITLE
Blog: Meet the Hoodies

### PR DESCRIPTION
**Pull request by gr2m**
I’m not sure if that’s the best title as we have the "Meet the Hoodies: {name}" series. But the events are called Meet the Hoodies, so don’t have a better idea ...

### CODE REVIEW

**Comment by NickColley** [comment]
"Meet the hoodies IRLsies"

**Comment by janl** [comment]
“Meet the Hoodies: Offline Edition”

**Comment by gr2m** [comment]
nice, I like "Offline Edition", fixed the title :)

**Comment by gr2m** [comment]
hold on, we will most likely do another Meet the Hoodies event in Washington D.C. on March 20th, should be settled early next week, if not we can post without announcing it :)

**Comment by janl** [comment]
your call :)

if we post this now, we can make a follow up when the DC meetup is fixed, roar blog posts yay :)

**Comment by varjmes** [comment]
I'd say post this and then do a follow up. So we can give the DC more attention? :)

File: _posts/2016-02-04-meet-the-hoodies-offline.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Maybe you’ve heard about Hoodie at a conference, or read about on Twitter, or through our initiatives like Offline First and noBackend and want to learn more, or you just want to get started with contributing to a meaningful Open Source project. Either way: **Meet the Hoodies is for you**!
+
+## Let’s meet the Hoodies!
+
+We would love to come visit your city and organise a fun workshop for new and experienced open source contributors, to give an introduction to Hoodie and above all: to meet you in person.
+
+While we love to see the Hoodie community grow, meeting you in person is what makes all the difference. These Github avatars are never going to be the same once you meet the friendly humans behind it – That goes both ways :)
+
+The setup is simple: we maintain a [list of great starter issues](http://go.hood.ie/hoodie-starter-issues) to make it as simple and fun as possible to start contributing to Hoodie. Attendees can choose any issue to work and and get all support you need from Hoodie hosts.
```
Comment (varjmes) [inline _posts/2016-02-04-meet-the-hoodies-offline.md]: 'Attendees can choose any issue to work' should be 'Attendees can choose any issue to work on'

File: _posts/2016-02-04-meet-the-hoodies-offline.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## New York City!
+
+[The first “Meet the Hoodies”](https://ti.to/hoodie/hello-nyc/) event happened in New York City on November 12th, hosted by [Gregor](http://twitter.com/gr2m) and [Nick](http://twitter.com/hipsterbrown).
+
+<a data-flickr-embed="true"  href="https://www.flickr.com/photos/116240988@N08/24146136463/" title="Meet the Hoodies NYC"><img src="https://farm2.staticflickr.com/1657/24146136463_176d3556fb_k.jpg" width="2048" height="1536" alt="Meet the Hoodies NYC"></a>
+
+And because it was so nice, [we came right back](https://ti.to/hoodie/hello-nyc-part-2/) on December 13, this time co-hosted by [Patrica](http://twitter.com/patggs).
```
Comment (varjmes) [inline _posts/2016-02-04-meet-the-hoodies-offline.md]: 'Patrica' should be 'Patricia' :)

File: _posts/2016-02-04-meet-the-hoodies-offline.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+<a data-flickr-embed="true" data-footer="true"  href="https://www.flickr.com/photos/116240988@N08/24146138643/in/photostream/" title="Meet the Hoodies NYC II"><img src="https://farm2.staticflickr.com/1716/24146138643_8749dc917f_k.jpg" width="2048" height="1536" alt="Meet the Hoodies NYC II"></a>
+
+<a data-flickr-embed="true"  href="https://www.flickr.com/photos/116240988@N08/24146136463/in/photostream/" title="Meet the Hoodies NYC"><img src="https://farm2.staticflickr.com/1657/24146136463_7f64b75722_b.jpg" width="1024" height="768" alt="Meet the Hoodies NYC"></a>
+
+Special thanks to the [Bocoup](https://bocoup.com/) and especially Brian for being such great hosts for our events.
```
Comment (varjmes) [inline _posts/2016-02-04-meet-the-hoodies-offline.md]: Is it 'the Bocoup' or 'Bocoup' ? I think it's just 'Bocoup'

File: _posts/2016-02-04-meet-the-hoodies-offline.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Berlin!
+
+We kicked of the year with [another “Meet the Hoodies”](https://ti.to/hoodie/berlin-meet-the-hoodies/) event, this time in Europe. A lot of Hoodies live or moved here, so it couldn’t take too long to organise another event here.
```
Comment (varjmes) [inline _posts/2016-02-04-meet-the-hoodies-offline.md]: 'We kicked of the year' should be 'We kicked off the year'

**Comment by varjmes** [comment]
@gr2m noticed a few little typos, hope it helps!

**Comment by gr2m** [comment]
@Charlotteis tremendously, thanks so much!!!

This is ready from my side. We have events for DC & Atlanta in the pipeline, but nothing available online yet.

**Comment by gr2m** [comment]
Thanks everyone <3

**Pull request merged**